gravitee-apim-portal-webui-next/src/app/api-details/api-tab-subscriptions/api-tab-subscriptions.component.scss

```diff
@@ -20,3 +20,17 @@
   justify-content: center;
   padding: 20px 0;
 }
+
+.api-tab-subscriptions__table {
+  overflow: hidden;
+  border: 1px solid #e6e0ec;
+  border-radius: 16px;
+
+  tr.mat-mdc-row.mdc-data-table__row {
+    background: #fff;
+  }
+
+  th.mat-mdc-header-cell {
+    font-weight: 600;
```

Comment:
Use one of the classes in `m3-adapter` to change the font style.

Reply:
Done